CHANGELOG.next.toml

```diff
@@ -79,3 +79,9 @@ message = "Expose local socket address from ConnectionMetadata."
 references = ["aws-sdk-rust#990"]
 meta = { "breaking" = false, "tada" = false, "bug" = false, "target" = "client" }
 author = "declanvk"
+
+[[smithy-rs]]
+message = "Optional implementation of Stream on PaginationStream"
+references = ["smithy-rs#3299"]
+meta = { "breaking" = false, "tada" = false, "bug" = false, "target" = "client"}
+author = "Erlend Langseth <erlend.langseth@openanalytics.eu>"
```

Comment:
we should probably also duplicate this under `aws-sdk-rust`, this will be relevant for folks using the SDK as well.

I think we should also change the message to be a little clearar about what folks can get and how, something like:

---

Add `PaginationStreamExt` extension trait to `aws-smithy-types-convert` behind the `convert-streams` feature. This makes it possible to treat a paginator as a [`futures_core::Stream`](https://docs.rs/futures-core/latest/futures_core/stream/trait.Stream.html), allowing customers to use stream combinators like [`map`](https://docs.rs/tokio-stream/latest/tokio_stream/trait.StreamExt.html#method.map) and [`filter`](https://docs.rs/tokio-stream/latest/tokio_stream/trait.StreamExt.html#method.filter).

Example: (double check this compiles)
```rust
use aws_smithy_types_convert::stream::PaginationStreamExt
s3_client.list_objects_v2().bucket("...").send().await.into_paginator().items().into_stream_03x()
```

---

Reply:
can I write all that in `message` with markdown formatting? Or maybe I can add that code example in rustdocs instead?

Reply:
yep, use triple quotes to surround your message `"""` and paste it in.

Reply:
not sure if you can easily grab the markdown out of my message, if not here: https://gist.github.com/rcoh/b69e8a42509842e239cd460b68053e16

Reply:
thanks, but a trick is to quote your message to get the markdown ;)